packages/components/src/TreeView/TreeViewItem/TreeViewItem.component.js

```diff
@@ -119,6 +120,7 @@ TreeViewItem.propTypes = {
 			label: PT.string,
 			icon: PT.string,
 		})),
+		counter: PT.number,
```

Comment:
Or not. 
No one in the Talend/ui repository uses PT as PropTypes.
Please align with the common style.